lib/scss/components/filter-menu/_filter-menu.scss

```diff
@@ -0,0 +1,118 @@
+@charset "UTF-8";
+
+//
+// Filter menu
+//
+
+// changing the layout for the filered pages
+@include grid-media-between('xs', 'lg') {
+  .left-region.flex-lg-3-of-12 {
+    flex: auto;
+    max-width: 100%;
+  }
+}
+
+.news-topics {
+  @include modular-spacing('padding', 3);
+
+  border: 1px solid $su-color-cool-grey-25;
+
+  .collapsable-menu {
+    @include type-e;
+
+    cursor: pointer;
+    display: inline-block;
+    text-decoration: none;
+    width: 100%;
+
+    @include grid-media('xl') {
+      @include modular-spacing('margin-bottom', 2);
+    }
+
+    &.show {
+      @include modular-spacing('margin-bottom', 2);
+
+      @include grid-media('xl') {
+        margin-bottom: 0;
+
+        &+ .menu > li.menu-item:last-of-type {
+          padding-bottom: 0;
+        }
+      }
+    }
+
+    .su-link--jump {
+      float: right;
+    }
+
+    @include grid-media-between('xs', 'lg') {
+      &:before {
+        @include small-paragraph;
+        @include padding(null 15px null null);
+
+        color: $su-color-bright-blue;
+        content: "\f1de"; // Placeholder Icon until SU ones in place.
+        font-family: 'Font Awesome 5 Free';
+        font-weight: 600; // Need this for the new version of FA. Will go away later.
+      }
+    }
+  }
+
+  .menu {
+    @include margin(0);
+    @include padding(0);
+
+    .menu-item {
+      @include modular-spacing('padding-bottom', 2);
+      @include small-paragraph;
+      @include modular-spacing('padding-left', 0);
+
+      a {
+        color: $su-color-black;
+        text-decoration: none;
+      }
```

Comment:
These items are missing hover and focus state. The focus state is actually the default blue bounding box and that is ok but there is no hover state.